package.json

```diff
@@ -47,6 +46,7 @@
     "eslint": "^5.13.0",
     "in-publish": "^2.0.0",
     "jest": "^19.0.2",
+    "loose-envify": "^1.4.0",
```

Comment:
this must be a runtime dependency, otherwise https://github.com/facebook/prop-types/issues/203#issuecomment-462398632 breaks people.

Reply:
> this must be a runtime dependency, otherwise [#203 (comment)](https://github.com/facebook/prop-types/issues/203#issuecomment-462398632) breaks people.

Got it, I thought it was only used for build in the first place.